<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTKGifLabelWindow</class>
 <widget class="QMainWindow" name="TTKGifLabelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TTKGifLabelWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TTKGifLabelWindow{background:rgba(50, 50, 50, 50);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <widget class="TTKGifLabelWidget" name="label_5">
      <property name="minimumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="TTKGifLabelWidget" name="label_2">
      <property name="minimumSize">
       <size>
        <width>58</width>
        <height>58</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>58</width>
        <height>58</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="TTKGifLabelWidget" name="label">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="TTKGifLabelWidget" name="label_3">
      <property name="minimumSize">
       <size>
        <width>26</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>26</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="TTKGifLabelWidget" name="label_4">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="TTKGifLabelWidget" name="label_6">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="TTKGifLabelWidget" name="label_7">
      <property name="minimumSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="TTKGifLabelWidget" name="label_8">
      <property name="minimumSize">
       <size>
        <width>93</width>
        <height>93</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>93</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="TTKGifLabelWidget" name="label_9">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="TTKGifLabelWidget" name="label_10">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TTKGifLabelWidget</class>
   <extends>QLabel</extends>
   <header>ttkgiflabelwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
